Sky Atlas/Sky Atlas.html

```diff
@@ -12299,8 +12299,8 @@ <h1>Active Data</h1>
             || JuliaChang | <a
               href="https://etherscan.io/address/0x252abAEe2F4f4b8D39E5F12b163eDFb7fac7AED7">https://etherscan.io/address/0x252abAEe2F4f4b8D39E5F12b163eDFb7fac7AED7</a>
             <a href="https://etherscan.io/verifySig/34622">https://etherscan.io/verifySig/34622</a> | <a
-              href="https://etherscan.io/address/0x7938b304F1c28e85b6A021251C8CeefF20370f38">https://etherscan.io/address/0x7938b304F1c28e85b6A021251C8CeefF20370f38</a>
-            <a href="https://etherscan.io/verifySig/34620">https://etherscan.io/verifySig/34620</a> | <a
+              href="https://etherscan.io/address/0xd0B44469d0f230FC67d2e6d2e8D239699eD76Bb5">https://etherscan.io/address/0xd0B44469d0f230FC67d2e6d2e8D239699eD76Bb5</a>
```

Comment:
```suggestion
              href="https://etherscan.io/address/0x5D4a96aFadb5505a8fF21ddeCa2Baf3A63301396">https://etherscan.io/address/0x5D4a96aFadb5505a8fF21ddeCa2Baf3A63301396</a>
```
It seems the same thing happened here, the delegate made a mistake and entered the address used to create the contract.